src/Eris/Generator/AssociativeArrayGenerator.php

```diff
@@ -14,7 +14,12 @@ function associative(array $generators)
 class AssociativeArrayGenerator implements Generator
 {
     private $generators;
-    
+
+    /**
+     * @var TupleGenerator
+     */
```

Comment:
however, the `/** @var /*` are outside of the conventions we used and should be removed (for what regards this project). The reason is that being PHP dynamic and being fields mostly `private`, they add weight without a relevant benefit.